### PR TITLE
JAMES-2260 RemoteDelivery retry delays configuration can contain spaces (community)

### PR DESCRIPTION
**Pull request by chibenwa**
Testing for https://github.com/apache/james-project/pull/89 .

Please review.

### CODE REVIEW

File: server/mailet/mailets/src/test/java/org/apache/james/transport/mailets/remoteDelivery/DelaysAndMaxRetryTest.java [hunk 1]
```diff
@@ -133,4 +133,11 @@ public void getExpendedDelaysShouldExpandMultipleDelays() throws Exception {
 
         assertThat(testee.getExpendedDelays()).containsExactly(1000L, 2000L, 2000L, 5000L, 5000L);
     }
+    
+    @Test
+    public void getExpendedDelaysShouldExpandMultipleDelaysWithSpaces() throws Exception {
```
Comment (mbaechler) [inline server/mailet/mailets/src/test/java/org/apache/james/transport/mailets/remoteDelivery/DelaysAndMaxRetryTest.java]: as you spotted the typo on Expended vs Expanded, maybe a separated commit with this refactoring would be a good idea ?

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**